test/src/e2e.dynval/2/snapshot.test.ts

```diff
@@ -17,8 +17,6 @@
 import * as chai from "chai";
 import { expect } from "chai";
 import * as chaiAsPromised from "chai-as-promised";
-import { SDK } from "codechain-sdk";
```

Comment:
I wonder why this line passed the linter.
It should've been marked as unused import.